src/main/java/core/basesyntax/OpationStrategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.OpationStrategy;
```

Comment:
There is a typo in the package name 'OpationStrategy'. It should likely be 'OperationStrategy'. Please correct the package name to maintain consistency and avoid issues with package imports.